packages/web-sdk/src/transports/otlp/payload/transform/transform.ts

```diff
@@ -0,0 +1,192 @@
+      toAttribute('grafana.browser.version', browser?.version),
+
+      toAttribute(SemanticResourceAttributes.TELEMETRY_SDK_NAME, sdk?.name),
+      toAttribute(SemanticResourceAttributes.TELEMETRY_SDK_VERSION, sdk?.version),
+      Boolean(sdk)
+        ? toAttribute(SemanticResourceAttributes.TELEMETRY_SDK_LANGUAGE, TelemetrySdkLanguageValues.WEBJS)
+        : undefined,
+
+      toAttribute(SemanticResourceAttributes.SERVICE_NAME, app?.name),
+      toAttribute(SemanticResourceAttributes.SERVICE_VERSION, app?.version),
+      toAttribute(SemanticResourceAttributes.DEPLOYMENT_ENVIRONMENT, app?.environment),
+    ].filter(isAttribute),
+  };
+}
+
+export function toScopeLog(transportItem: LogTransportItem): ScopeLog {
+  return {
+    scope: {
+      name: '@grafana/faro-web-sdk',
+      version: VERSION,
+    },
+    logRecords: [toLogRecord(transportItem)],
+  };
+}
+
+function toLogRecord(transportItem: LogTransportItem): LogRecord {
+  const { type } = transportItem;
+
+  switch (type) {
+    case TransportItemType.LOG:
+      return toLogLogRecord(transportItem as TransportItem<LogEvent>);
+    case TransportItemType.EXCEPTION:
+      return toErrorLogRecord(transportItem as TransportItem<ExceptionEvent>);
+    case TransportItemType.EVENT:
+      return toEventLogRecord(transportItem as TransportItem<EventEvent>);
+    case TransportItemType.MEASUREMENT:
+      return toMeasurementLogRecord(transportItem as TransportItem<MeasurementEvent>);
+    default:
+      const error = `Unknown TransportItemType: ${type}`;
+      internalLogger.error(error);
+      throw new Error(error);
+  }
+}
+
+function toLogLogRecord(transportItem: TransportItem<LogEvent>): LogRecord {
+  const { meta, payload } = transportItem;
+  const timeUnixNano = toTimeUnixNano(payload.timestamp);
+  const body = toAttributeValue(payload.message) as { stringValue: string; key: string };
+
+  return {
+    timeUnixNano,
+    severityNumber: 10,
+    severityText: 'INFO2',
+    body,
+    attributes: getCommonLogAttributes(meta),
+    traceId: payload.trace?.trace_id,
+    spanId: payload.trace?.trace_id,
+  } as const;
+}
+
+function toEventLogRecord(transportItem: TransportItem<EventEvent>): LogRecord {
+  const { meta, payload } = transportItem;
+  const timeUnixNano = toTimeUnixNano(payload.timestamp);
+  const body = toAttributeValue(payload.name) as { stringValue: string; key: string };
+
+  return {
+    timeUnixNano,
+    body,
+    attributes: [
+      ...getCommonLogAttributes(meta),
+      toAttribute('event.name', payload.name), // No prefix because this is a semantic attribute. But event.name constant is currently missing in sematic-conventions npm package
+      toAttribute('event.domain', payload.domain), // No prefix because this is a semantic attribute. But event.domain constant is currently missing in sematic-conventions npm package
+      toAttribute('grafana.event.attributes', payload.attributes),
```

Comment:
if we're gonna scope it, might make more sense to use `faro.event.attributes`, this is "faro" sdk after all.
BUT: why not just inline them as `event.[attr key]` each? I think this is what I'd expect as a user

Reply:
So this is intended to be an unopinionated API for pushing events, not semantically Faro specific. I think it should allow providing arbitrary event attributes, and it should be up to the caller to decide on the attribute name. Eg, an instrumentation may want to provide a valid semantic OTEL attribute, but would be unable to because it is automatically prefixed. If we have Faro instrumentations creating Faro specific attributes, the instrumentation should be responsible for naming, not the core SDK. Does that make sense?

Reply:
After a call with @domasx2 we decided to:
- Remove the prefixes from most attributes
- For some very Faro specific ones like `faro.error.stacktrace`, which has it's own data structure, we keep the prefix but change it to Faro.